Source/JavaScriptCore/wasm/WasmIPIntSlowPaths.h

```diff
@@ -58,8 +58,11 @@ WASM_IPINT_EXTERN_CPP_HIDDEN_DECL(epilogue_osr, CallFrame* callFrame);
 #endif
 
```

Comment:
Ditto. Should we attach SYSV_ABI for WASM_IPINT_EXTERN_CPP_DECL related functions?